functions/Get-DbaBackupInformation.ps1

```diff
@@ -53,6 +53,12 @@ function Get-DbaBackupInformation {
     .PARAMETER PassThru
         When data is exported the cmdlet will return no other output, this switch means it will also return the normal output which can be then piped into another command
     
+    .PARAMETER MaintenanceSolution
+        This switch tells the function that the folder is the root of a Ola Hallengren backup folder
+
+    .PARAMETER IgnoreLogBackup
```

Comment:
Just to have clarity on the docs, can you add an example for these additional parameters @Stuart-Moore

Reply:
Just pushed up a version with examples for those.